src/Raven.Studio/wwwroot/Content/css/icons.less

```diff
@@ -2,6 +2,7 @@
 @arrow-right: "\f115";
 @icon-document: "\f11c";
 @trash: "\f11d";
+@conflict: "\f160";
```

Comment:
I think you need to fix this codepoint, see: https://github.com/ravendb/ravendb/commit/ce1b1f21e27e6739d7dd98c25a40621b1485f398

Reply:
done